examples/show_file_contents.php

```diff
@@ -2,10 +2,10 @@
 
 require_once __DIR__ . '/../vendor/autoload.php';
 
-use GitElephant\Objects\Branch;
-use GitElephant\Objects\Commit;
-use GitElephant\Objects\Tag;
-use GitElephant\Repository;
+use \GitElephant\Objects\Branch;
```

Comment:
Interesting ) which IDE you are using for editing? just curious..
Actually it's a default code-style for declaring used namespaces, i.e. without leading backslash.

P.S. In case you are interested in code-style, I would recommend to view php code standards PSR-2 and PSR-12. Most modern php ide support auto-codestyling or phpcs can be used with preset settings for PSRs

Reply:
I did this change manually (or, well, Regexp replace in VSCode search) just to have it uniform across all files. I decided in this cases for use of the "\" in front because this was the one used in the majority of the files and attributed it to a decision by Matteo. But I agree, it seemed uncommon. I remove it everywhere in the next commit. 
Yes, I know about phpcs – in the GitHub actions, we actually test for PSR-2 (as this is the one Matteo mentioned in the REAME), required to pass all tests (see composer script [`check-cs`](https://github.com/matteosister/GitElephant/blob/9c2be09ed540df868035ef12b6d1f44d6b90e0c8/composer.json#L6)). Do you want me to add PSR-12 too? Any other style to add to the check?